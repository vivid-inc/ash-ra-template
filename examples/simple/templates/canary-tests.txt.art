Minimum supported Clojure version: <%= (clojure-version) %>
