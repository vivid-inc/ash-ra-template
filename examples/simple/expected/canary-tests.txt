Minimum supported Clojure version: 1.9.0
